src/COWFeeModule.sol

```diff
@@ -8,19 +8,16 @@ import { GPv2Order } from "./libraries/GPv2Order.sol";
 
 IGPv2Settlement constant settlement = IGPv2Settlement(0x9008D19f58AAbD9eD0D60971565AA8510560ab41);
 address constant vaultRelayer = 0xC92E8bdf79f0507f65a392b0ab4667716BFE0110;
```

Comment:
Set `GPv2Settlement` via the constructor, and expose a public immutable variable, in addition to the `vaultRelayer` + `domainSeparator`.- both of which can be inferred from the settlement contract.